### ISSUE TITLE
open root folder using side menu

### ISSUE THREAD

**Issue by schaarsc** [nextcloud/android#3219]
### Actual behaviour
- app stays in same folder, only title is changed to "Nextcloud"

### Expected behaviour
- open root folder
 
### Steps to reproduce
1. open app, change into /some/folder
2. open side menu and select "all files"

### Environment data
Android version: 7
Nextcloud app version: 3.3.1

**Comment by tobiasKaminsky** [comment]
Correct, this only changes the view for current folder from "on device" to "all files".
So you only see a change when you previously clicked on "on device".

Having some sort of "home" button, which navigates to root would be nice.

@nextcloud/designers @AndyScherzinger on webUI we have a bread crumb navigation.
On android there is too little space for this.

So I have two ideas:
- "on device" -> "all files" -> show all files in current folder.
- already "all files" -> clicking "all files" again -> to root

or 
- implement some sort of navigation like we already have at upload:
![image](https://user-images.githubusercontent.com/5836855/47995181-c665c300-e0f4-11e8-8dc7-3cc77fc5dfb8.png)

**Comment by tobiasKaminsky** [comment]
Hm, this could be done in addition.
If you are 5 or even 10 folders deep and want to go to a specific parent folder you would have to swipe many times, like it also now.

Btw: is swiping a common/well know navigation pattern on Android?

**Comment by Michele64** [comment]
Swiping is like flipping book pages, and you can navigate single handed
I saw it in the Dropbox app and it was as quite intuitive way to change folders (as long as I used it; -)

**Comment by tobiasKaminsky** [comment]
@AndyScherzinger what is your opinion here?

**Comment by jancborchardt** [comment]
> already "all files" -> clicking "all files" again -> to root

@tobiasKaminsky this is definitely good.

The swiping seems nice too. Haven’t seen it anywhere except the Dropbox app, but hierarchies are also specific to apps like ours. Could be very nice, and even swiping a few times is quite simple.

**Comment by joshtrichards** [comment]
Not really relevant today and even less so with #14146
